### ISSUE TITLE
Power litmit, what is long,short and time window ?

### ISSUE THREAD

**Issue by ForteDexe** [georgewhewell/undervolt#188]
Sorry I am new, I am not sure what it means.

**Comment by WinWeirdo** [comment]
Powerlimit is the internal limit that blockes power being drawn over the limit. long and short are the names of the two, one is short as for short duration and the other one is long as for longer duration. The time window is the amount of the seconds from being activited to stop limiting that amount and returning to normal powerlimit or shorter one

**Comment by mathew2214** [comment]
is there a way to disable power limits entirely? let the system run as hard as it needs to for however long it needs to?

**Comment by WinWeirdo** [comment]
Yes, just set powerlimit to 999 and that would "disable" powerlimit.

**Comment by mathew2214** [comment]
just seems to set the limits to 999, and not actually disable them. they still read as enabled.

**Comment by WinWeirdo** [comment]
You technically cannot disable powerlimit, only set it to impossible to achive value like 9999 W, no processor in existence would survive that much power.

**Comment by mathew2214** [comment]
yeah, it doesnt work. i set it to something that ill never actually hit, and throttling occurs anyways
